crates/wit-parser/src/errors.rs

```diff
@@ -34,8 +34,12 @@ pub enum WITParserError {
     ITRemainderNotEmpty,
 
     /// An error occurred while parsing WIT section.
-    #[error("IT section is corrupted")]
-    CorruptedITSection,
+    #[error(
+        "IT section is corrupted: {0}.\
+    \nProbably the module was compiled with an old of fce cli, please try to update and recompile.\
```

Comment:
Probably the module was compiled with an old version of fce cli, please update and recompile.